rust/.openapi-generator-ignore

```diff
@@ -30,8 +30,5 @@
 /.gitlab-ci.yml
 /.travis.yml
 /src/lib.rs
-/src/apis/client.rs
-/src/apis/configuration.rs
-/src/apis/request.rs
```

Comment:
Do you know why these no longer exist? Maybe there's new files now that we should be skipping.

Reply:
I checked that these files were not produced by the generator after I removed these lines. I'm not sure what the initial reason was for these entries. As for new files being generated instead, I'll double check to make sure we're not generating files that have no corresponding `mod`, but nothing jumped out at first glance.

I don't think it would hurt anything to retain these if we want, other than the possible confusion of a discussion like this later.

Reply:
> I checked that these files were not produced by the generator after I removed these lines.

I may have been remembering something similar in another lib. These files _are produced_ by the generator, but

```
$ git log -- src/apis/{client,configuration,request}.rs
```

gives nothing back, so I don't think these files ever existed in git.

Looking closer for references to these modules:

```
$ git grep -e 'mod \(client\|config\|request\)' ./src/
src/lib.rs:mod request;
```

only one `mod request` shows up, and it points to the top-level `request.rs` rather than the one that would be produced by the generator.

---

Okay, good catch. 👍🏻 

I now think the ignore entries are there to avoid producing files that will not be referenced by the rest of the source tree. I'll add these entries back in, thus ensuring we see breakage at build time in the case one of the other modules starts trying to reference them unexpectedly.